app/services/form_response.rb

```diff
@@ -19,6 +20,11 @@ def to_h
     hash = { success: success }
     hash[:errors] = errors if !serialize_error_details_only?
     hash[:error_details] = flatten_details(error_details) if error_details.present?
+    pii_keys.each do |key|
+      hash[:pii_like_keypaths] ||= []
+      hash[:pii_like_keypaths].append([:error_details, key]) if error_details.has_key?(key)
+      hash[:errors] = hash[:errors].except(key) if hash[:errors].has_key?(key)
+    end
```

Comment:
The goal for `pii_like_keypaths` was that it's passed in at the `analytics.track_event` callsite, is the reason we're doing this here because for these 2 forms, the analytics happens "automatically" inside of the FSM?

Reply:
Yes, the FSM decouples the form from the analytics so we don't directly control how the errors are logged.

Reply:
Ah I just found how we work around this in other places, we add it to the `extra` part of the `FormResponse`:
https://github.com/18F/identity-idp/blob/main/app/forms/idv/address_form.rb#L27

Can we try that approach instead?

Reply:
This is a case where we should avoid logging the error message, which could in some cases expose a person's name. The same approach doesn't seem acceptable here.

Reply:
If the goal is to not log them, can we remove them in the form before instantiating the FormResponse? FormResponse is to capture and transport data specifically to our logs

Reply:
Like in this spot: https://github.com/18F/identity-idp/blob/2469f2a4aaef534b7d165c2a34e621db247f7f4d/app/services/idv/steps/in_person/address_step.rb#L52

we could still submit the form, but then post-process that `FormResponse` to omit errors we don't want to log

Reply:
e.g. create a version of the `Errors` object with the transliteration errors filtered out when creating `FormResponse` in these classes?

Reply:
yup exactly that

Reply:
Ok, that seems doable. Thanks!